### DISCUSSION TITLE
If you need a full unlock dm me

### DISCUSSION THREAD

**Discussion by jhonnyonthespot** [3kh0/ext-remover#1084] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Please read before dming, I have fully unlocked chromebooks and am willing to share how I did it, the Chromebooks i have gotten fully unlocked to chrome os with dev mode and no enterprise enrollment is, 100e, 100e 2nd gen, 100e 2nd gen mtk, 100e 2nd gen amd, 100e 3rd gen and 3rd gen mtk. For all of these chromebooks it does require opening it up and taking out motherboard and could brick your Chromebook if you do it wrong. If you want to dm me and I'll show you how.

**Comment by VFXKLIPS** [comment]
> Please read before dming, I have fully unlocked chromebooks and am willing to share how I did it, the Chromebooks i have gotten fully unlocked to chrome os with dev mode and no enterprise enrollment is, 100e, 100e 2nd gen, 100e 2nd gen mtk, 100e 2nd gen amd, 100e 3rd gen and 3rd gen mtk. For all of these chromebooks it does require opening it up and taking out motherboard and could brick your Chromebook if you do it wrong. If you want to dm me and I'll show you how.

How do I dm on github

or ad me on discord it is ramy_meta

**Reply by AshtonDavies** [reply]
[Pencil Method](https://chrose.netlify.app/detail/Pencil%20Method)
